Tests/ClientTests/Helpers/NotificationManagerTests.swift

```diff
@@ -0,0 +1,103 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+@testable import Client
+
+class NotificationManagerTests: XCTestCase {
+    private var center: MockUserNotificationCenter!
+    private var notificationManager: NotificationManager!
+
+    override func setUp() {
```

Comment:
If you have a setUp you should have a tearDown as well to nil your test variables